unit_tests/test_tlslite_extensions.py

```diff
@@ -22,7 +22,7 @@
         RenegotiationInfoExtension, ALPNExtension, StatusRequestExtension, \
         SupportedVersionsExtension, VarSeqListExtension, ListExtension, \
         ClientKeyShareExtension, KeyShareEntry, ServerKeyShareExtension, \
-        CertificateStatusExtension
+        CertificateStatusExtension, HRRKeyShareExtension
```

Comment:
the doc string for that class is
```
Class for handling the Hello Retry Request variant of the Key Share ext.
```
It may not be obvious from the name, but I'd say the necessary comment already exists...